### ISSUE TITLE
MacOS support

### ISSUE THREAD

**Issue by cosmincatalin** [dotnet/spark#107]
Please add support for running in MacOS.

**Comment by imback82** [comment]
Thanks for reporting.

If you are not using UDFs in your application, you can follow [Ubuntu instruction](https://github.com/dotnet/spark/blob/master/docs/getting-started/ubuntu-instructions.md) to get it working; you can skip installing the Microsoft.Spark.Worker.

If you are using UDFs in your appliction, you can follow [this](https://github.com/dotnet/spark/blob/master/docs/getting-started/ubuntu-instructions.md) to build the Microsoft.Spark.Worker from source.

In short, instead of
```
dotnet publish -f netcoreapp2.1 -r ubuntu.18.04-x64
```
, you will do
```
dotnet publish -f netcoreapp2.1 -r osx-x64
```

We plan to have Microsoft.Spark.Worker for MacOS in upcoming releases. Stay tuned!

**Comment by cosmincatalin** [comment]
Thanks for the guidance. Works fine. Maybe the documentation needs to be updated with this.
